VTEX - Orders API.json

```diff
@@ -94,98 +94,105 @@
+                                            "description": "Order creation date in [ISO 8601 time zone offset format](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DDThh:mm:ssZ`."
                                         },
                                         "lastChange": {
                                             "type": "string",
-                                            "description": "Order's last change date in [ISO 8601 time zone offset format](https://learn.microsoft.com/pt-br/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DD hh:mm:ss[.nnnnnnn]+hh:mm`."
+                                            "description": "Order last change date in [ISO 8601 time zone offset format](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DDThh:mm:ssZ`."
                                         },
                                         "orderGroup": {
                                             "type": "string",
-                                            "description": "Order's group ID."
+                                            "description": "Order group ID, a code part of the order ID that groups all orders related to the same purchase. For example, in order ID `v71021570str-02`), the order group ID is `v71021570str`."
                                         },
                                         "followUpEmail": {
                                             "type": "string",
-                                            "description": "Email of the store's employee responsible for managing the order."
+                                            "description": "Email of the store employee responsible for managing the order."
                                         },
                                         "lastMessage": {
                                             "type": "string",
                                             "nullable": true,
-                                            "description": "Transactional message sent most recently."
+                                            "description": "Last transactional message sent to the customer."
                                         },
                                         "hostname": {
                                             "type": "string",
-                                            "description": "Account host name, which is the [domain](https://help.vtex.com/en/tutorial/configuring-the-store-domain--tutorials_2450#store-address-structure) registered in **Account Settings**."
+                                            "description": "Main [account name](https://help.vtex.com/en/tutorial/what-is-an-account-name--i0mIGLcg3QyEy8OCicEoC)."
```

Comment:
Is this really only the account name (e.g. `apiexamples`) or the full hostname (e.g. `www.apiexamples.com`)?

Asking because of the name of the parameter and the previous description including this doc https://help.vtex.com/en/tutorial/configuring-the-store-domain--tutorials_2450#store-address-structure